tracer/src/Datadog.Trace/Tagging/GrpcTags.cs

```diff
@@ -37,8 +37,8 @@ public GrpcTags(string spanKind)
         [Tag(Trace.Tags.GrpcMethodService)]
         public string MethodService { get; set; }
 
-        [Tag(Trace.Tags.GrpcStatusCode)]
-        public string StatusCode { get; set; }
+        [Metric(Trace.Tags.GrpcStatusCode)]
+        public double? StatusCode { get; set; }
```

Comment:
Can we instead make this a nullable int?

Reply:
I believe the source generator expects a double IIRC.

Reply:
Ah you're right, resolving